### PR TITLE
Zone editing

### PR DESCRIPTION
**Pull request by huffman**

### CODE REVIEW

**Comment by huffman** [comment]
Having an issue with overlay visibility that I am trying to debug...

**Comment by hifi-gustavo** [comment]
Build console output for commit: 4a99c21e7bf5d6a2ed563590cedf035411fd8542 available at: https://gist.github.com/9ff3504a38d06252c556<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4677.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4677.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 30b9e16c1a0b1cba9d7618e2297243fc041c9625 available at: https://gist.github.com/9ff3504a38d06252c556<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4677.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4677.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 78207f0d519a112fa1ac1d1ace2f7ae51136c0dc available at: https://gist.github.com/9ff3504a38d06252c556<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4677.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4677.dmg

File: interface/src/ui/overlays/Cube3DOverlay.cpp [hunk 1]
```diff
@@ -77,7 +77,7 @@ void Cube3DOverlay::render(RenderArgs* args) {
                         if (_drawOnHUD) {
                             DependencyManager::get<GeometryCache>()->renderSolidCube(1.0f, glm::vec4(1.0f, 1.0f, 1.0f, alpha));
                         } else {
-                            DependencyManager::get<DeferredLightingEffect>()->renderSolidCube(1.0f, glm::vec4(1.0f, 1.0f, 1.0f, alpha));
+                            DependencyManager::get<GeometryCache>()->renderSolidCube(1.0f, glm::vec4(1.0f, 1.0f, 1.0f, alpha));
```
Comment (huffman) [inline interface/src/ui/overlays/Cube3DOverlay.cpp]: This is temporary until we can actually fix transparency in the next sprint.  Transparency doesn't work with the deferred lighting pipeline.

**Comment by hifi-gustavo** [comment]
Build console output for commit: 361bf7e4413ba0c2e72326b408fb7a7194be70df available at: https://gist.github.com/e0d7cb53a8c15a5777b5<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4677.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4677.dmg

**Pull request merged**